### ISSUE TITLE
[CI] DockerTests test171AdditionalCliOptionsAreForwarded failing

### ISSUE THREAD

**Issue by valeriy42** [elastic/elasticsearch#88581]
Authorization fails on multiple linux distributions with the same error message
```
java.lang.RuntimeException: Request failed:	
HTTP/1.1 401 Unauthorized	
{"error":{"root_cause":[{"type":"security_exception","reason":"unable to authenticate user [elastic] for REST request [/]","header":{"WWW-Authenticate":["Basic realm=\"security\" charset=\"UTF-8\"","Bearer realm=\"security\"","ApiKey"]}}],"type":"security_exception","reason":"unable to authenticate user [elastic] for REST request [/]","header":{"WWW-Authenticate":["Basic realm=\"security\" charset=\"UTF-8\"","Bearer realm=\"security\"","ApiKey"]}},"status":401}	
```

**Build scan:**
https://gradle-enterprise.elastic.co/s/lje4c3us2kdnk/tests/:qa:os:destructiveDistroTest.default-dockerCloud/org.elasticsearch.packaging.test.DockerTests/test171AdditionalCliOptionsAreForwarded

**Reproduction line:**
`null`

**Applicable branches:**
master

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.packaging.test.DockerTests&tests.test=test171AdditionalCliOptionsAreForwarded

**Failure excerpt:**
```
java.lang.RuntimeException: Request failed:
HTTP/1.1 401 Unauthorized
{"error":{"root_cause":[{"type":"security_exception","reason":"unable to authenticate user [elastic] for REST request [/]","header":{"WWW-Authenticate":["Basic realm=\"security\" charset=\"UTF-8\"","Bearer realm=\"security\"","ApiKey"]}}],"type":"security_exception","reason":"unable to authenticate user [elastic] for REST request [/]","header":{"WWW-Authenticate":["Basic realm=\"security\" charset=\"UTF-8\"","Bearer realm=\"security\"","ApiKey"]}},"status":401}

  at __randomizedtesting.SeedInfo.seed([2F89EBE6ED525BDE:E8E51550E20BEB57]:0)
  at org.elasticsearch.packaging.util.ServerUtils.makeRequest(ServerUtils.java:359)
  at org.elasticsearch.packaging.util.docker.Docker.getJson(Docker.java:632)
  at org.elasticsearch.packaging.test.DockerTests.test171AdditionalCliOptionsAreForwarded(DockerTests.java:1106)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:577)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
  at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:833)

```

**Comment by pugnascotia** [comment]
Interesting - I can replicate this after all, if I test the Cloud image instead of the default image. Investigating...

**Comment by pugnascotia** [comment]
Fixed by #88584.
